source/developers-guide/plugins/models/index.md

```diff
@@ -0,0 +1,701 @@
+---
```

Comment:
Most, if not all of this file, covers Doctrine basic stuff that is already covered in the Doctrine docs. Please rewrite this to only include links to the official docs + Shopware specific info